README.md

```diff
@@ -97,6 +97,32 @@ Example codes:
 ./examples/print_as_pdb ../mmtf_spec/test-suite/mmtf/173D.mmtf
 ```
 
+## Benchmark
+
+using the following simple code:
+```cpp
+#include <mmtf.hpp>
+
+
+int main(int argc, char** argv)
+{
+        for (int i=1; i<argc; ++i) {
+                mmtf::StructureData sd;
+                mmtf::decodeFromFile(sd, argv[i]);
+        }
+}
+```
+compiled via:
+```
+g++ -Ofast -Immtf/include  -Imsgpack/include  decode_all_mmtf.cpp
+```
+
+We are able to load 153,987 mmtf files (current size of the pdb) from an SSD in (n=4)
```

Comment:
Does it make sense to have 4 timings or should we just report 1 number as the average of 4 separate runs? It's a fairly consistent timing so I think a single average is enough.

Also can you report the total size in MB of the MMTF files?

Reply:
up to you i don't really care that much. But sometimes disk access can be cached so you have to do multiple.

Reply:
```suggestion
We are able to load 153,987 mmtf files (current size of the pdb) or  14.3GB from an SSD in (n=4)
```